contrib/packaging/bootupctl-update.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
Let's name this unit `bootloader-update.service` as its what will be user visible and users ideally should not have to know about bootupd.